config/testdata/conf.good.yml

```diff
@@ -23,7 +23,7 @@ scrape_configs:
   # metrics_path defaults to '/metrics'
   # scheme defaults to 'http'.
 
-  file_sd_configs:
+  sd_files:
     - names:
```

Comment:
I think Brian was referring to the "names" field in the issue rather than "file_sd_configs".